### PR TITLE
updating the contents in `statistical_variable.md` and adding a reference StatVar Explorer tool

### PR DESCRIPTION
**Pull request by sharadshriram**
The content displayed under the `Statistical Variable` menu in the docsite seems to render a subset of StatisticalVariables that was hand-curated as a list. In this PR, we replace the list of StatVar categories and update the contents of the `statistical_variable.md` file to point a link to the [`Stat Var Explorer`](https://datacommons.org/tools/statvar) tool.

After the proposed change is merged, the `statistical_variable.md` page will be rendered as shown below:

### Based on review
image

### Before review
image.

### CODE REVIEW

**Review by beets — COMMENTED** [review]
thanks for sending this pr!

since the diff is too large to add comments to, adding them here. many nitpick comments - since this is public documentation, want to make sure we get the details right.

- please add a note saying that the current list of all stat vars are no longer available (please use the stat var tool which is always kept up to date)
- no need to say "statvar for short" -- we never use that term in the docs
- please always use "statistics" and not "stats"
- watch for capitalization of "data commons"
- i'm not sure if adding an example mcf is helpful to API users. they probably want to know how to find the stat var.
- typo in link to browser node (perhaps we should link to the tool as well?)
- instead of geographic level, they will be looking for that stat for a place

think about this from the perspective of a developer or analyst wanting to use the API. feel free to borrow language from the glossary or from the stat var tool intro page.

File: statistical_variables.md [hunk 1]
```diff
@@ -6,21 +6,20 @@ nav_order: 4
...
 
-```
-Node: dcid:Count_Person_Male
-typeOf: dcs:StatisticalVariable
-statType: dcs:measuredValue
-populationType: schema:Person
-gender: dcs:Male
-measuredProperty: dcs:count
-```
+Using the [StatVar Explorer tool](https://datacommons.org/tools/statvar), it is easy for you to get an overview of the different data sources in which the [number of males in a population](https://datacommons.org/browser/Count_Person_Male) is observed as a data point. The tool also enables you to view the available observations made for the StatisticalVariable at different places with example observation time-series to visualize and counts of the number of places for which the statistics are available.
```
Comment (beets) [inline statistical_variables.md:18]:
```suggestion
The [Statistical Variable Explorer](https://datacommons.org/tools/statvar) tool makes it easy for you to get information about each variable, such as the different data sources, metadata and places for which the statistics are available.
```

File: statistical_variables.md [hunk 2]
```diff
@@ -6,21 +6,20 @@ nav_order: 4
...
 
-To explore the different Statistical Variables in Data Commons, please visit the [StatVar Explorer tool](https://datacommons.org/tools/statvar)
+The StatVar Explorer tool is also helpful to find exisiting StatisticalVariable definitions which you can re-use in your data import. In the context of the [example StatisticalVariable](https://datacommons.org/browser/Count_Person_Male), you search for it using search terms like "Person Male" and the result will have all the StatisticalVariables that are avaialble on the data commons knowledge graph containing the `populationType = Person` and `gender = Male`. The [example StatisticalVariable](https://datacommons.org/browser/Count_Person_Male) can be accessed by the name of the StatisticalVariable, in this case `Male Population` from the hierarchy of StatisticalVariables (on left-pane of the page) occuring for the `Person With Gender = Male` result.
```
Comment (beets) [inline statistical_variables.md:20]: This and the following paragraph is too much information for this page; some of it belongs in the tool. We just want to lead users to the tool from here.

**Review by beets — COMMENTED** [review]
Thanks for the updates. Might be easier to iterate on this in a doc but getting close.

File: _layouts/default.html [hunk 3]
```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="icon" href="/assets/images/favicon.ico" type="image/x-icon"/>
   <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
[the remark this answers is not in the document]
Comment (sharadshriram) [reply _layouts/default.html:10]: Got it! I have shifted from `font-awesome` to `material icons`

File: statistical_variables.md (hunk 2 above)
Comment (sharadshriram) [reply statistical_variables.md:20]: Got it! I have made a call to action from the user with a button to redirect them to the tool! 
So, the page will look like this.
image

File: _layouts/default.html [hunk 4]
```diff
@@ -7,6 +7,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="icon" href="/assets/images/favicon.ico" type="image/x-icon"/>
   <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined">
```
Comment (beets) [inline _layouts/default.html:11]: You don't need this -- just the one above on line 10

File: _sass/base.scss [hunk 5]
```diff
@@ -50,6 +50,10 @@ md: 720px,
 lg: 960px
 );
 
+/** Icon sizes **/
+.material-icons.md-16 { font-size: 16px; }
+.material-icons-outlined.md-16 { font-size: 16px; }
```
Comment (beets) [inline _sass/base.scss:55]: Ditto - can remove line 55

File: glossary.md [hunk 6]
```diff
@@ -14,9 +14,10 @@ This page contains a selection of key terms important to understanding the struc
 
 A group of entities sharing some characteristic. Interchangeably referred to in a Data Commons context as `Cohort` and `CohortSet`. Examples include [the CDC's list of the United States' 500 largest cities](https://datacommons.org/browser/CDC500_City).
 
->  **NOTE:**
->
->  The type [`Cohort`](https://datacommons.org/browser/Cohort) is a legacy type not used by the Sheets method `DCCOHORTMEMBERS()`.
+<div markdown="span" class="alert alert-success" role="alert">
```
Comment (beets) [inline glossary.md:17]:
```suggestion
<div markdown="span" class="alert alert-info" role="alert">
```

File: _sass/base.scss [hunk 7]
```diff
@@ -50,6 +50,10 @@ md: 720px,
 lg: 960px
 );
 
+/** Icon sizes **/
+.material-icons.md-16 { font-size: 16px; }
```
Comment (beets) [inline _sass/base.scss:54]: try this. note the formatting of the css as well 

```suggestion
.material-icons.md-16 { 
  font-size: 16px;
  padding-right: 8px;
  vertical-align: middle;
}
```

**Review by beets — CHANGES_REQUESTED** [review]
thanks for the update. the screenshots are helpful!

File: glossary.md (hunk 6 above)
Comment (sharadshriram) [reply glossary.md:17]: Done.

File: _layouts/default.html (hunk 4 above)
Comment (sharadshriram) [reply _layouts/default.html:11]: Ah! Sorry, I was checking out `material-icons-outlined` but forgot to remove them before checking in the code.
I have cleaned this up now! 😄

File: _sass/base.scss (hunk 5 above)
Comment (sharadshriram) [reply _sass/base.scss:55]: Done.

**Review by beets — APPROVED** [review]
thanks for the updates -- and for contributing to our documentation! these changes will be pushed automatically once it's merged.

**Pull request merged**